common/src/java/com/zimbra/common/soap/SyncConstants.java

```diff
@@ -157,4 +157,7 @@ public final class SyncConstants {
     public static final String ZIMBRASYNC_CMD_MOVE_ITEMS = "MoveItems";
     public static final String ZIMBRASYNC_CMD_SMART_REPLY = "SmartReply";
     public static final String ZIMBRASYNC_CMD_SMART_FORWARD = "SmartForward";
+    
+    // App specific
+    public static final String BOXER_APP = "Boxer";
```

Comment:
is just Boxer Or BoxerImageApp?

Reply:
the String that is coming for userAgent is like below
`AirWatch Boxer (Apple iPod touch; iOS 15.7.1) Version 22.11.0/9703`
so I've added as Boxer